pyanaconda/modules/payloads/payload/dnf/tree_info.py

```diff
@@ -158,14 +158,18 @@ def _load_tree_info(self, root_path, file_path=None, file_content=None):
                 tree_info.load(file_path)
 
             tree_info.validate()
+            log.debug("Loaded treeinfo metadata:\n%s", tree_info.dumps())
 
             # Load the release version.
             release_version = tree_info.release.version.lower()
 
-            # Load the repositories.
+            # Create repositories for variants and optional variants.
+            # Child variants (like addons) will be ignored.
             repo_list = []
 
             for name in tree_info.variants:
```

Comment:
@jkonecny12, your implementation of the treeinfo support skips child variants (like addons). Is it ok?

Reply:
It is.
Reason for that is that no-one requested the support + it shouldn't be used. The addons were used on RHEL-7 .treeinfo and for RHEL-8 it was decided to avoid using addons but rather go with the complete variants.

Reply:
Thanks for the confirmation!